include/z64bgcheck.h

```diff
@@ -27,10 +24,30 @@ struct DynaPolyActor;
 #define FUNC_80041EA4_VOID_OUT 12
 
 typedef struct {
-    Vec3f scale;
-    Vec3s rot;
-    Vec3f pos;
-} ScaleRotPos;
+    /* 0x00 */ Vec3f scale;
+    /* 0x0C */ Vec3s rot;
+    /* 0x14 */ Vec3f pos;
+} ScaleRotPos; // size = 0x20
+
+// Macros for `CollisionPoly`
+
+#define COLPOLY_NORMAL_FRAC (1.0f / SHT_MAX)
+#define COLPOLY_SNORMAL(x) ((s16)((x) * SHT_MAX))
+#define COLPOLY_GET_NORMAL(n) ((n)*COLPOLY_NORMAL_FRAC)
+#define COLPOLY_VTX_CHECK_FLAGS_ANY(vIA, flags) ((vIA) & (((flags)&7) << 13))
```

Comment:
Not sure if a generic macro for both set of flags is appropriate, but maybe it is. Additionally the first arg is still named `vIA` but is now used to access data packed with all vertex indices.